packages/docs/src/styles/_sections.scss

```diff
@@ -263,11 +263,15 @@
 .docs-overlay-example-transition {
   $overlay-example-width: $pt-grid-size * 40;
 
-  position: absolute;
+  position: fixed;
   top: $overlay-example-width / 2;
   left: calc(50% - #{$overlay-example-width / 2});
   z-index: $pt-z-index-overlay + 1;
   width: $overlay-example-width;
+
+  .pt-overlay-inline & {
```

Comment:
Indeed, users shouldn't have to explicitly style their overlay content, we should handle that automatically for them